package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "commercetools-sunrise-data",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)",
+  "main": "index.js",
+  "scripts": {
+    "set:authUrl": "npm config set commercetools-sunrise-data:authUrl",
+    "set:apiUrl": "npm config set commercetools-sunrise-data:apiUrl",
+    "set:projectKey": "npm config set commercetools-sunrise-data:project",
+    "set:clientId": "npm config set commercetools-sunrise-data:client_id",
+    "set:clientSecret": "npm config set commercetools-sunrise-data:client_secret",
+    "get:config": "echo \"authUrl\": && npm config get commercetools-sunrise-data:authUrl && echo \"apiUrl\": && npm config get commercetools-sunrise-data:apiUrl && echo \"projectKey\": && npm config get commercetools-sunrise-data:project && echo \"clientId\": && npm config get commercetools-sunrise-data:client_id && echo \"clientSecret\": && npm config get commercetools-sunrise-data:client_secret",
+    "build": "rimraf dist/ && babel ./**/*.js --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
+    "credentials": "npm run build && node dist/lib/update-credentials.js --project $npm_package_config_project --client_id $npm_package_config_client_id --client_secret $npm_package_config_client_secret --apiUrl $npm_package_config_apiUrl --authUrl $npm_package_config_authUrl",
+    "clean:data": "./mvnw spring-boot:run -DprojectCleanUp=true -DprojectSetUp=false -DchannelsImport=false -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
+    "import:projectSetup": "./mvnw spring-boot:run -DprojectCleanUp=false -DprojectSetUp=true -DchannelsImport=true -DcatalogImport=false -DreserveInStore=false -DordersImport=false -Dctp.projectKey=$npm_package_config_project -Dctp.clientId=$npm_package_config_client_id -Dctp.clientSecret=$npm_package_config_client_secret -Dctp.authUrl=$npm_package_config_authUrl -Dctp.apiUrl=$npm_package_config_apiUrl",
```

Comment:
Any chance to start the java parts with loglevel WARN?   INFO is spamming so much that my IntelliJ console buffer was not big enough to see whether earlier steps had errors.

Reply:
Yes, I think we can just update the logback.xml and that would do the job.
Update it to `WARN` ?

Reply:
sounds good.   choose the level yourself after trying out.

Reply:
Done! commercetools namespaces will still output on INFO, but the default spring/java is set to warn.

9e56fd5